alfresco-googledrive-repo-base/src/main/java/org/alfresco/integrations/google/docs/GoogleDocsModel.java

```diff
@@ -1,41 +1,39 @@
 /**
  * Copyright (C) 2005-2015 Alfresco Software Limited.
```

Comment:
Take the opportunity to update all licence headers for consistency, eg. 2005-2019 .. albeit soon to be 2020 ;-)

Reply:
Done. I've changed it to 2020.